### PR TITLE
Improve grid styles

### PR DESCRIPTION
**Pull request by NateWr**
This PR:

- Switches to the lighter grid header styles I passed around the UI/UX group.
- Adds text label to the grid filter search toggle
- Removes extra space below grid filter submission button
- Fixes alignment issues experienced in __some__ grids when no items exist.

@beghelli This last change required removing an empty `<tbody>` in `grid.tpl`. I suspect it was there for a reason (perhaps some kind of JS-only insertion?). But I wasn't able to see any problems introduced with this change. Take a look and let me know what you think.

### CODE REVIEW

File: templates/controllers/grid/grid.tpl [hunk 1]
```diff
@@ -74,8 +74,6 @@
 		{/if}
 		{foreach from=$gridBodyParts item=bodyPart}
 			{$bodyPart}
-		{foreachelse}
-			<tbody><tr></tr></tbody>
```
Comment (beghelli) [inline templates/controllers/grid/grid.tpl]: I think this is here for the category grids with no category items, so it will be ready to receive any new item that could be added and retrieved by Ajax. But I am checking :)

File: templates/controllers/grid/grid.tpl [hunk 2]
```diff
@@ -83,7 +81,7 @@
 				so that we can restore it if the user deletes all rows.
 			**}
 			<tr>
-				<td colspan="{$grid->getColumnsCount('indent')|@count}">{translate key=$grid->getEmptyRowText()}</td>
```
Comment (beghelli) [inline templates/controllers/grid/grid.tpl]: I think this will break the category grids, checking that also

Comment (NateWr) [inline templates/controllers/grid/grid.tpl]: I believe that category grids have [their own template](https://github.com/pkp/pkp-lib/blob/master/templates/controllers/grid/gridBodyPartWithCategory.tpl#L30) where an empty `<tbody>` is created. This change adjusts the regular grids to match the category grid technique, but maybe I missed something.

Comment (beghelli) [inline templates/controllers/grid/grid.tpl]: This is just for the body. The grid.tpl file is used by both, Grid and CategoryGrid controllers.

**Comment by NateWr** [comment]
@beghelli Have you had a chance to check on whether or not this PR breaks the category grids?

**Comment by NateWr** [comment]
@beghelli Just another reminder on this PR.

**Comment by beghelli** [comment]
Thanks @NateWr, checked the category grids and everything is fine.

**Comment by NateWr** [comment]
Sweet! Been wanting to look at some lighter styles. :)
